USAGE.md

```diff
@@ -0,0 +1,66 @@
+# carpentries-theme
+
+This repository provides a remote Jekyll theme for use with The Carpentries.
+
+> !!! CAUTION !!!
+>
+> This is *early alpha* work, not yet ready for wider adoption.
+
+Many lessons built for The Carpentries carry around the website theme assets
+such as CSS and favicons. These things evolve, but keeping a given lesson
+synchronized with upstream repositories (such as [styles](
+https://github.com/carpentries/styles) and [lesson-example](
```

Comment:
technically lesson example is not an upstream repository for the styling: it demonstrates how to use the styles repository